### ISSUE TITLE
Not updated on NPM Package

### ISSUE THREAD

**Issue by SaffaMeer** [bodunadebiyi/react-magnific-popup#5]
Please update the npm package on npm site. It is not updated.

**Comment by Hess-Gregory** [comment]
Yes, correct, I am also looking to install it on my React project. It is the redesign of a website originally in html which uses this dependency a little from everywhere, I should certainly migrate to another dependency because it seems that this one is abandoned by its editor
